types/geakstr__sass-inline-svg/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "esModuleInterop": true,
```

Comment:
Please remove this line. Most tsconfigs these days have `"esModuleInterop": true`, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.